gradle/fips.gradle

```diff
@@ -0,0 +1,55 @@
+import org.elasticsearch.gradle.ExportElasticsearchBuildResourcesTask
+import org.elasticsearch.gradle.testclusters.ElasticsearchCluster
+import org.elasticsearch.gradle.testclusters.ElasticsearchCluster
+
+// Common config when running with a FIPS-140 runtime JVM
+if (Boolean.parseBoolean(System.getProperty("tests.fips.enabled"))) {
```

Comment:
We should use `BuildParams.inFipsJvm` here instead.

Reply:
Done